pkg/webhook/server.go

```diff
@@ -280,6 +288,19 @@ func (s *Server) Start(ctx context.Context) error {
 	return nil
 }
 
+// StartedChecker returns an healthz.Checker which is healthy after the
+// server has been started.
+func (s *Server) StartedChecker() healthz.Checker {
```

Comment:
I can also move this func into the healthz package. The consequence would be that either the healthz package has a dependency onto the webhook package or I would have to add an interface with a `Started() bool` func.

Reply:
Could we maybe also tcp ping the server?

Reply:
Yup definitely. Should we then still keep the started flag? 

Or is the idea to combine that, so we only start pinging after started is true?

I'm not sure if it's worth it keeping the started field around.

Reply:
It can make sense to only start tcp-pinging the port only after the started field is set to true

Reply:
Yup, will do.